### ISSUE TITLE
some questions

### ISSUE THREAD

**Issue by gengyanlei** [GuYuc/WS-DAN.PyTorch#6]
Hello, author.
I encountered the following problems while looking at your code:
1. optimizer.zero_grad location should not be placed with loss.back
2. When calculating accuracy, the return should be np.array(), not tensor type
The understanding is wrong, please point out, thank you.

**Comment by GuYuc** [comment]
@gengyanlei Hi. Thanks for suggestions. 
1. I suppose this would not affect our training because gradients for this batch were calculated in the next step. The forward process would not calculate gradients.
2. The return type of accuray is np.array(), and our metrics are np.array().
